samples/apps/copilot-chat-app/scripts/README.md

```diff
@@ -0,0 +1,48 @@
+# Copilot Chat Setup Scripts
+
+> The PowerShell scripts in this directory require [PowerShell Core 6 or higher](https://github.com/PowerShell/PowerShell#get-powershell).
+
+## Before You Begin
+To run these scripts, you will need the following:
+- *YOUR_CLIENT_ID*
+  - This is the client ID (also known as application ID) associated with your Azure Active Directory (AAD) application registration, which you can find in the Azure portal.
+- *YOUR_TENANT_ID*
+  - This is the tenant ID (also known as directory ID) associated with your AAD app registration, which you can find in the Azure portal. If you are using a personal MSA account, you can simply enter `msa` for this value to use the personal MSA tenant.
```

Comment:
"If you are using a personal MSA account"...  I don't know whether most people outside Microsoft know what MSA means.  Maybe replace that with "If you a using a personal (non work or school) Microsoft account"?

Reply:
+1

Reply:
I'm questioning if "tenant ID" is the right identifier here. Would "audience" be better? I don't want to use "authority" because not asking them to provide the whole URL, just the last part (which could be a tenant ID or something else)

Reply:
AAD uses the terms "Directory ID" and "Tenant ID" for things like the tenant guids, "common", etc. The link at least shows the possibilities and whay they mean,. "Audience" in the OAuth/AAD token world is used as a general term for the application ID.

Reply:
Ok I'm definitely confused about what "audience" is... I'll just use tenant ID.